### PR TITLE
Correct description of use of variables in example

### PR DESCRIPTION
**Pull request by matthurne**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/CloudWatch-Events-Input-Transformer-Tutorial.md [hunk 1]
```diff
@@ -58,4 +58,4 @@ We achieve this by mapping the *instance* variable to the `$.detail.state` JSON
 
 1. Choose **Configure details**\.
 
-1. Type a name and description for the rule, and choose **Create rule**\.
\ No newline at end of file
+1. Type a name and description for the rule, and choose **Create rule**\.
```
Comment (matthurne) [inline doc_source/CloudWatch-Events-Input-Transformer-Tutorial.md:61]: No idea why this line is showing as modified.

Comment (davidmc24) [reply doc_source/CloudWatch-Events-Input-Transformer-Tutorial.md:61]: Looks like you added a trailing newline character.

Comment (matthurne) [reply doc_source/CloudWatch-Events-Input-Transformer-Tutorial.md:61]: I see. I used GitHub's direct file edit feature, so I blame that. :-)

Comment (chrisdr) [reply doc_source/CloudWatch-Events-Input-Transformer-Tutorial.md:61]: Thanks, mhurne, good catch.

**Comment by chrisdr** [comment]
Merging in mhurne's example variable correction.

**Pull request merged**